flink-runtime/src/main/java/org/apache/flink/runtime/dispatcher/DispatcherRestEndpoint.java

```diff
@@ -120,7 +120,12 @@ public DispatcherRestEndpoint(
 				// register extension handlers
 				handlers.addAll(webSubmissionExtension.getHandlers());
 			} catch (FlinkException e) {
-				log.info("Failed to load web based job submission extension.", e);
+				if (log.isDebugEnabled()) {
+					log.debug("Failed to load web based job submission extension.", e);
+				} else {
+					log.info("Failed to load web based job submission extension. " +
```

Comment:
It still sounds like an error. Maybe `Web based job submission is disabled. Probable reason: [...]`